tests/integration_tests/bugs/test_lp1898997.py

```diff
@@ -0,0 +1,71 @@
+"""Integration test for LP: #1898997
+
+cloud-init was incorrectly excluding Open vSwitch bridge members from its list
+of interfaces.  This meant that instances which had only one interface which
+was in an Open vSwitch bridge would not boot correctly: cloud-init would not
+find the expected physical interfaces, so would not apply network config.
+
+This test checks that cloud-init believes it has successfully applied the
+network configuration, and confirms that the bridge can be used to ping the
+default gateway.
+"""
+import pytest
+
+MAC_ADDRESS = "de:ad:be:ef:12:34"
+
+
+NETWORK_CONFIG = """\
+bridges:
+        ovs-br:
+            dhcp4: true
+            interfaces:
+            - enp5s0
+            macaddress: 52:54:00:d9:08:1c
+            mtu: 1500
+            openvswitch: {{}}
+ethernets:
+    enp5s0:
+      mtu: 1500
+      set-name: enp5s0
+      match:
+          macaddress: {}
+version: 2
+""".format(MAC_ADDRESS)
+
+
+@pytest.mark.lxd_config_dict({
+    "user.network-config": NETWORK_CONFIG,
+    "volatile.eth0.hwaddr": MAC_ADDRESS,
+})
+@pytest.mark.lxd_vm
```

Comment:
lxd_container should work here too, right?

Reply:
I'll double-check, but IIRC OVS pokes at the kernel in a way that isn't permitted from (non-privileged) containers.

Reply:
Yep, they fail to come back up once `openvswitch` is installed:

```
$ lxc console --show-log intimate-mullet

Console log:

         Starting Cleanup of Temporary Directories...
[  OK  ] Finished Cleanup of Temporary Directories.
[***   ] A start job is running for OpenVSwi… for ovs-br (19min 44s / no limit)

```